src/status_im/android/core.cljs

```diff
@@ -127,13 +126,13 @@
                                :chat-list main-tabs
                                :new-chat new-chat
                                :new-group new-group
-                               :edit-group edit-group
+                               :edit-group edit-contact-group
```

Comment:
Do we want consistency across keywords too? e.g. `new-group-chat`? Should keywords themselves be updated?